Jenkinsfile

```diff
@@ -225,15 +245,17 @@ pipeline {
         )
         string(name: 'KUBECTL_VERSION',
                description: 'kubectl version',
-               defaultValue: '1.21.5'
```

Comment:
Why we are downgrading the KUBECTL_VERSION from 1.21.5 to 1.25.4 ?

Reply:
This is an upgrade from 1.21.5 to 1.25.4.